extensions/transaction/WFS3core+STAC+extensions.yaml

```diff
@@ -0,0 +1,1008 @@
+  
+        If a feature has multiple temporal properties, it is the decision of the
+        server whether only a single temporal property is used to determine the
+        extent or all relevant temporal properties.
+      example: '2018-02-12T00:00:00Z/2018-03-18T12:31:12Z'
+    itemCollection:
+      type: object
+      required:
+        - features
+        - type
+      properties:
+        type:
+          type: string
+          enum:
+            - FeatureCollection
+        features:
+          type: array
+          items:
+            $ref: '#/components/schemas/item'
+        links:
+          $ref: '#/components/schemas/itemCollectionLinks'
+    item:
+      type: object
+      required:
+        - id
+        - type
+        - geometry
+        - links
+        - properties
+        - assets
+      properties:
+        id:
+          type: string
+          example: path/to/example.tif
+          description: 'unique ID, potentially link to file'
+        bbox:
+          $ref: '#/components/schemas/bbox'
+        geometry:
+          type: object
+          description: footprint of the item
+        type:
+          type: string
+          description: The GeoJSON type
+          enum:
+            - Feature
+        properties:
+          $ref: '#/components/schemas/itemProperties'
+        links:
+          type: array
+          items:
+            $ref: '#/components/schemas/link'
+        assets:
+          type: object
+          additionalProperties:
+            type: object
+            required:
+              - href
+            properties:
+              href:
+                type: string
+              name:
+                type: string
+      example:
+        type: Feature
+        id: CS3-20160503_132130_04
+        bbox:
+          - -122.59750209
+          - 37.48803556
+          - -122.2880486
+          - 37.613537207
+        geometry:
+          type: Polygon
+          coordinates:
+            - - - -122.308150179
+                - 37.488035566
+              - - -122.597502109
+                - 37.538869539
+              - - -122.576687533
+                - 37.613537207
+              - - -122.2880486
+                - 37.562818007
+              - - -122.308150179
+                - 37.488035566
+        properties:
+          datetime: '2016-05-03T13:21:30.040Z'
+          provider: 'http://www.cool-sat.com'
+          license: CC-BY-4.0
+        links:
+          - rel: self
+            href: >-
+              http://cool-sat.com/catalog/collections/cs/items/CS3-20160503_132130_04.json
+        assets:
+          analytic:
+            name: 4-Band Analytic
+            href: >-
+              http://cool-sat.com/catalog/collections/cs/items/CS3-20160503_132130_04/analytic.tif
+          thumbnail:
+            name: thumbnail
+            href: >-
+              http://cool-sat.com/catalog/collections/cs/items/CS3-20160503_132130_04/thumb.png
+    partialItem:

```

Comment:
How does partialItem differ from Item? Is it just removing the required portion? I think that would be covered by calling out to the rfc I suggested above and we can just document Item once?